json-form-react/JsonForm.js

```diff
@@ -53,22 +53,51 @@ const JsonForm = (props) => {
     const fieldClasses = `jsfr-field ${field.class} ${formErrors[field.name] ? 'jsfr-field-error' : ''}`;
     const fieldWrapperClasses = `jsfr-field-wrapper ${field.wrapperClass} ${formErrors[field.name] ? 'jsfr-field-wrapper-error' : ''}`;
 
-    return (
-      <div key={field.name} className={fieldWrapperClasses}>
-        <label htmlFor={fieldId}>
-          {field.label}
-        </label>
-
-        <input
-          id={fieldId}
-          type={field.type}
-          name={field.name}
-          value={formData[field.name] || ''}
-          placeholder={field.placeholder}
-          className={fieldClasses}
-          onChange={onChange}/>
-      </div>
-    );
+    if (field.type === 'radio') {
+      return (
+        <div key={field.name} className={fieldWrapperClasses}>
+          <label htmlFor={fieldId}>
+            {field.label}
+          </label>
+          
+          {Object.keys(field.option).map((keys) => {
+            return [
+              <>
```

Comment:
`key` attribute needs to be set here, not on the label. React will look for the `key` on the main component that comes inside a `map()` function. Also please set the `key={index}` like I said before.

Reply:
Sure @cie6868 I have added an index there. and I use labels for showing the name of the radio button.